src/integrationTest/java/eu/xenit/gradle/testrunner/Reproductions.java

```diff
@@ -125,6 +126,7 @@ public void testIssue104() throws IOException {
     }
 
     @Test
+    @Ignore //Waiting for a fix for https://github.com/bmuschko/gradle-docker-plugin/issues/1158
```

Comment:
(non-blocking) If you write `@Ignore("Waiting for a fix for https://github.com/bmuschko/gradle-docker-plugin/issues/1158")` it also gets printed to the test report.